.goreleaser.yml

```diff
@@ -5,7 +5,7 @@ builds:
     flags:
       - -trimpath
     ldflags:
-      - "-s -w -X main.version={{.Version}} -X main.commit={{.Commit}}"
+      - "-s -w -X github.com/tailscale/terraform-provider-tailscale/tailscale.providerVersion={{.Version}} -X main.commit={{.Commit}}"
```

Comment:
This needs to be the fully qualified path since it no longer lives in `main.go`. See [here](https://github.com/hashicorp/terraform-provider-azurerm/blob/75d6f38bd5c92cdd4825d5a5324b5dfc26cc7104/.goreleaser.yml#L38) in the AzureRM Terraform provider for similar usage.